cg/meta/report/report_api.py

```diff
@@ -289,15 +291,15 @@ def get_workflow_accreditation_limitation(self, application_tag: str) -> str | N
         )
         return application_limitation.limitations if application_limitation else None
 
-    def get_sample_application_data(self, lims_sample: dict) -> ApplicationModel:
+    def get_sample_application_data(self, sample: Sample, lims_sample: dict) -> ApplicationModel:
         """Retrieves the analysis application attributes."""
         application: Application = self.status_db.get_application_by_tag(
             tag=lims_sample.get("application")
```

Comment:
can't we get this one also from the sample with `Sample.application_version.application`? So that we remove the lims_sample from here

Reply:
No, we can't. LIMS is more reliable for this step. Sometimes we only make changes in LIMS, like when we need more reads but don't want to change the invoicing.

https://atlas.scilifelab.se/production/lab/supporting_information/sample_registration/#41-changing-application-tag-for-a-sample